prog-o-meter.py

```diff
@@ -146,7 +152,39 @@ def add_day(self):
         update_days_file(self.filename, self.days)
         self.canvas.itemconfig(self.countdown_text, text = "".join(("You have ", str(self.days_remaining), " days left!\n\n", "If you code everyday, you will be done with this project on ", self.completion_date)))
         if self.days >=self.GOAL:        # Disable add_day_button if goal have been reached 
-            self.add_day_button.config(state = "disabled")             
+            self.add_day_button.config(state = "disabled")  
+    def log_entry(self):
+        """Opens a new window for user to make a new log entry. The user can make any number of entries he/she wishes.
+
+        Callback function to add_log_button. Opens a new toplevel window with a text widget and three buttons:
+        A button marked "Save" that saves the text the user typed into the widget to the [USERNAME]_log.txt file.
+        A button marked "Clear" that clears the text that the user typed into the widget.
+        A button marked "Close" that closes the toplevel window.
+        """
+        t = Tk.Toplevel(self.root)

```

Comment:
Just for simplicity, as our codebase keeps expanding, can we give t a more descriptive name?

Reply:
"t" has been changed to "log_window", which I hope is more descriptive.